fuel-vm/src/interpreter/contract.rs

```diff
@@ -154,6 +154,26 @@ where
         .ok_or_else(|| PanicReason::ContractNotFound.into())
 }
 
+/// Gets the bytes of the contract code from storage. If the contract is longer that
+/// specified `size` the contract will be shortened to `size` bytes. If the contract is
+/// shorter than `size` bytes, then the contract will stay it's original length.
+pub(crate) fn contract_bytes<S>(
```

Comment:
I might suggest keeping the name `contract`, since the return type is still a `Contract`. To me, `contract_bytes` implies a vector of bytes or something along those lines.

Reply:
There is a separate function called `contract`. I had it named `contract_partial` before, but that's less clear IMO.

Reply:
What about something like `contract_subsection`?

Reply:
Cool, sounds good to me.